drivers/include/hih6130.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2015 Eistec AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    driver_hih6130 HIH6130 humidity and temperature sensor
+ * @ingroup     drivers
+ * @brief       Device driver for Honeywell HumidIcon Digital
+ *              Humidity/Temperature Sensors: HIH-6130/6131 Series
+ * @{
+ *
+ * @file
+ * @brief       Device driver for Honeywell HumidIcon Digital
+ *              Humidity/Temperature Sensors: HIH-6130/6131 Series
+ *
+ * @author      Joakim Gebart <joakim.gebart@eistec.se>
+ */
+
+#ifndef HIH6130_H_
+#define HIH6130_H_
+
+#include <stdint.h>
+
+#include "periph/i2c.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Device descriptor for HIH6130/HIH6131 sensors
+ */
+typedef struct {
+    i2c_t i2c;              /**< I2C device the sensor is connected to */
+    uint8_t addr;           /**< the slave address of the sensor on the I2C bus */
+} hih6130_t;
+
+/**
+ * @brief Initialize a sensor
+ *
+ * @param[out] dev          device descriptor of sensor to initialize
+ * @param[in]  i2c          I2C bus the sensor is connected to
+ * @param[in]  address      I2C slave address of the sensor
+ *
+ * @return                  0 on success
+ * @return                  <0 on error
+ */
+int hih6130_init(hih6130_t *dev, i2c_t i2c, uint8_t address);
+
+/**
+ * @brief Read humidity from sensor and convert to floating-point percent
+ *
+ * @param[in]  dev                       Sensor device descriptor
+ * @param[out] relative_humidity_percent Measured relative humidity in percent
+ *
+ * @return 0 on success
+ * @return -1 on error
+ * @return 1 if data is stale
+ */
+int hih6130_get_humidity_float(hih6130_t *dev, float *relative_humidity_percent);
+int hih6130_get_humidity_temperature_float(hih6130_t *dev,
```

Comment:
No documentation for that function? I would have assumed to have (a) just this one function that reads both values or (b) also to have a separate function to get the temperature value.